src/market.py

```diff
@@ -113,42 +117,42 @@ def base_size_number_to_lots(self, size: float) -> int:
     @staticmethod
     def get_mint_decimals(endpoint: str, mint_pub_key: PublicKey) -> int:
         """Get the mint decimals from given public key."""
-        bytes_data = _load_bytes_data(mint_pub_key, endpoint)
+        bytes_data = load_bytes_data(mint_pub_key, endpoint)
         return MINT_LAYOUT.parse(bytes_data).decimals
 
     def load_bids(self):
         """Load the bid order book"""
         bids_addr = PublicKey(self._decode.bids)
-        bytes_data = _load_bytes_data(bids_addr, self._endpoint)
+        bytes_data = load_bytes_data(bids_addr, self._endpoint)
         return OrderBook.decode(self, bytes_data)
 
     def load_asks(self):
         """Load the Ask order book."""
         asks_addr = PublicKey(self._decode.asks)
-        bytes_data = _load_bytes_data(asks_addr, self._endpoint)
+        bytes_data = load_bytes_data(asks_addr, self._endpoint)
         return OrderBook.decode(self, bytes_data)
 
     def load_event_queue(self):
```

Comment:
Missing type hint.

Reply:
It's probably meaningless to add type hint here. Since we are still using the raw `construct` which is not typed. And same below.